action.yml

```diff
@@ -67,13 +73,13 @@ runs:
       run: |
         if npm run | grep -q "test:unit" ; then

```

Comment:
We test for existence of the unit test script here. This needs to be modified aswell. Maybe leave the `npm run` out of the inputs and just specify `test:unit` there?